index.js

```diff
@@ -13,7 +13,8 @@
 module.exports = function (api, options) {
   api.assertVersion(7);
 
-  const isWebpack = (options && options.webpack);
+  const isWebpack = (options && options.webpack) || api.env('webpack');
+  const isReactServer = (options && options.reactServer) || api.env('reactServer');
```

Comment:
Any reason to keep both methods of lookup? This is a major version bump, so if it's cleaner to look for api.env, just do that.

Reply:
Webpack uses options, but `@gasbuddy/react` uses `BABEL_ENV`, so it was for supporting both styles.